industry_benchmarks/utils/info_form_private_sets.txt

```diff
@@ -0,0 +1,87 @@
+#################################
+### Private dataset info form ###
+#################################
+
+System preparation
+------------------
+
+1. Protein preparation
+
+a) Structure model type, e.g. X-ray, cryo-em, homology model:
+-
+
+b) Structure preparation details, e.g. modelling of missing loops, assignment of protonation states:
+-
+
+
+2. Ligand preparation
+
+a) Method for assigning the protonation state:
+-
+
+b) Method for choosing the tautomer (if applicable):
+-
+
+c) Known binding mode: [ ] Yes  [ ] No
+
+
+d) Method used for generating the binding mode (e.g. (constrained) docking, alignment on co-crystalized ligand):
+-
+
+
+3. Known binding site: [ ] Yes  [ ] No
+
+
+Benchmark difficulty
+--------------------
+
+1. Estimated benchmark difficulty
+
+[ ] Easy
+[ ] Medium
+[ ] Hard
+
+2. Potential challenges
+
+[ ] Protonation states ligands
+[ ] Protonation states protein
+[ ] Water sampling
+[ ] Ions in the binding site
+[ ] Metals in the binding site
+[ ] Known sampling issues
+    [ ] Ligands
+    [ ] Protein
+[ ] Diverse chemistry / known force field parameterization issues
+[ ] Poor ligand alignment
+[ ] Large perturbations
+[ ] ...
+
+3. Additional details for the system specific challenges:
+-
+
+
+Experimental assay
+------------------
+
+1. Broad assay description (e.g. “Kd from ITC”):
+-
+
+2. Assay error provenance, e.g. multiple repeats:
```

Comment:
```suggestion
8. Assay error provenance, e.g. multiple repeats (if applicable):
```
Could this reveal the underlying assay type? If so we should make this optional.